modules/payloads/singles/ruby/pingback_bind_tcp.rb

```diff
@@ -34,17 +34,14 @@ def generate
 
   def ruby_string
     self.pingback_uuid ||= self.generate_pingback_uuid
-
-    return "require 'socket';"+
-      "s=TCPServer.new(#{datastore['LPORT'].to_i});"+
-      "c=s.accept;"+
-      "s.close;"+
-      "c.puts(\'#{self.pingback_uuid.gsub('-','')}\'.scan(/../).map { |x| x.hex.chr }.join);"+
-      "c.close"
- end
-
-  def include_send_pingback
-    true
+    ruby_code = <<~RUBY
```

Comment:
Using the squiggly heredoc here adds extra newlines which the original payload was able to omit. Since this is a payload, style of the contents doesn't matter as much as finished size.